### ISSUE TITLE
Outdated Documentation

### ISSUE THREAD

**Issue by CiaranWelsh** [dpilger26/NumCpp#80]
I've just started using this package and I noticed that the documentation is outdated in at least one place. Namely, [here](https://dpilger26.github.io/NumCpp/doxygen/html/index.html). It seems that 
```
nc::random::randn<double>(nc::Shape(3,4))
```
should be 
```
nc::random::randN<double>(nc::Shape(3,4))
```
Great work on a cool package, this looks like it will be useful - thanks.

**Comment by dpilger26** [comment]
Thanks, I'll get that fixed in the next release.

**Comment by dpilger26** [comment]
Fixed in version 2.4.0 release.
